cli/src/parse.rs

```diff
@@ -39,11 +42,104 @@ impl fmt::Display for Stats {
     }
 }
 
+#[derive(Debug, Copy, Clone)]
+pub struct ParseTheme {
+    pub node_kind: Option<Color>,
+    pub node_text: Option<Color>,
+    pub field: Option<Color>,
+    pub token: Option<Color>,
+    pub row_color: Option<Color>,
+    pub row_color_named: Option<Color>,
+    pub extra: Option<Color>,
+    pub error: Option<Color>,
+}
+
+impl ParseTheme {
+    #[must_use]
+    pub const fn blank() -> Self {
```

Comment:
I think `empty` or `none` fit better, up to you

```suggestion
    pub const fn empty() -> Self {
```